### PR TITLE
[docs] CDCSDK: Add documentation for transaction ordering with CDC

### PR DESCRIPTION
**Pull request by vaibhav-yb**
This PR adds the documentation related to using transaction ordering with CDC in YugabyteDB.

### CODE REVIEW

File: docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md [hunk 1]
```diff
@@ -1110,6 +1111,40 @@ However, some sink connectors may not understand the preceding format. `PGCompat
...
+| transforms.Reroute.type | Transformer class to be used. |
+| transforms.Reroute.key.field.regex | Specifies a regular expression that the transformation applies to the default destination topic name to capture one or more groups of characters. |
+| transforms.Reroute.key.field.replacement | Specifies a regular expression for determining the value of the inserted key field in terms of those captured groups. |
+| provide.transaction.metadata | Whether to generate events with transaction boundaries. |
+
+For usage example, refer to our [example repository](https://github.com/yugabyte/cdc-examples/tree/main/consistent-streaming).
+
+### Transaction boundaries
+
+The connector publishes metadata which can be used to distinguish transaction boundaries for a downstream application to implement atomicity. Once the configuration property `provide.transaction.metadata` (see [transaction metadata]()) is enabled, the connector will also publish events indicating the beginning and end of the transaction.
```
Comment (ddhodge) [inline docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md:1139]:
```suggestion
The connector publishes metadata that can be used to distinguish transaction boundaries for a downstream application to implement atomicity. Once the configuration property `provide.transaction.metadata` is enabled, the connector will also publish events indicating the beginning and end of the transaction. For more information, see [Transaction metadata](#transaction-metadata).
```

File: docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md [hunk 2]
```diff
@@ -1110,6 +1111,40 @@ However, some sink connectors may not understand the preceding format. `PGCompat
...
+
+### Prerequisites
+* Stream ID should be created in the `EXPLICIT` checkpointing mode. To know more, see [yb-admin create\_change\_data_stream](../../admin/yb-admin#create_change_data_stream).
```
Comment (ddhodge) [inline docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md:1142]:
```suggestion
* Create the Stream ID should in the `EXPLICIT` checkpointing mode. For more information, see [yb-admin create\_change\_data_stream](../../../admin/yb-admin#create_change_data_stream).
```

File: docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md [hunk 3]
```diff
@@ -1110,6 +1111,40 @@ However, some sink connectors may not understand the preceding format. `PGCompat
...
+* The connector should always be run with a single task i.e. `tasks.max` should always be set to 1.
```
Comment (ddhodge) [inline docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md:1143]:
```suggestion
* You should always run the connector with a single task, that is, `tasks.max` should always be set to 1.
```

File: docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md [hunk 4]
```diff
@@ -1110,6 +1111,40 @@ However, some sink connectors may not understand the preceding format. `PGCompat
...
+
+### Known limitations with transaction ordering
```
Comment (ddhodge) [inline docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md:1145]:
```suggestion
### Known limitations
```

File: docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md [hunk 5]
```diff
@@ -1110,6 +1111,40 @@ However, some sink connectors may not understand the preceding format. `PGCompat
...
+* Transactional ordering is currently not supported with schema evolution. Issue [18476](https://github.com/yugabyte/yugabyte-db/issues/18476).
```
Comment (ddhodge) [inline docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md:1146]:
```suggestion
* Transactional ordering is currently not supported with schema evolution. See issue [18476](https://github.com/yugabyte/yugabyte-db/issues/18476).
```

File: docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md [hunk 6]
```diff
@@ -1110,6 +1111,40 @@ However, some sink connectors may not understand the preceding format. `PGCompat
...
+| provide.transaction.metadata | Whether to generate events with transaction boundaries. |
+
+For usage example, refer to YugabyteDB CDC Consistent Streaming Pipeline in the [example repository](https://github.com/yugabyte/cdc-examples/tree/main/consistent-streaming).
+
+### Transaction boundaries
+
+The connector publishes metadata that can be used to distinguish transaction boundaries for a downstream application to implement atomicity. Once the configuration property `provide.transaction.metadata` is enabled, the connector will also publish events indicating the beginning and end of the transaction. For more information, see [Transaction metadata](#transaction-metadata).
+
+### Prerequisites
+* Create the Stream ID should in the `EXPLICIT` checkpointing mode. For more information, see [yb-admin create\_change\_data_stream](../../../admin/yb-admin#create_change_data_stream).
```
Comment (ddhodge) [inline docs/content/preview/explore/change-data-capture/debezium-connector-yugabytedb.md:1142]:
```suggestion
* Create the Stream ID should in the `EXPLICIT` checkpointing mode. For more information, see [yb-admin create\_change\_data_stream](../../../admin/yb-admin#create-change-data-stream).
```

**Review by ddhodge — APPROVED** [review]
Should this go in Stable as well?

**Comment by vaibhav-yb** [comment]
Yes, I will add the content to the stable section as well.

**Review by ddhodge — APPROVED** [review]

**Pull request merged**